HACKING.md

```diff
@@ -26,6 +28,40 @@ And lastly get Webpack and the development dependencies:
     $ npm install
     $ ./node_modules/bower/bin/bower install
 
+When relying on CI to run the test suite, this is all that is
+necessary to work on the JavaScript components of Cockpit.
+
+To actually build the Cockpit binaries themselves from source
+(including to run the integration tests locally), you will need
+additional header files and other components. Check
+`tools/cockpit.spec` for the concrete Fedora build dependencies.
+The following should work in a fresh Git clone:
+
+    $ sudo yum-builddep tools/cockpit.spec
+
+In addition, for testing, the following dependencies are required:
+
+    $ sudo yum install \
+        curl \
+        expect \
+        libvirt \
+        libvirt-client \
+        libvirt-daemon \
+        libvirt-python \
+        python \
+        python-libguestfs \
+        python-lxml \
+        qemu \
+        qemu-kvm \
+        rpm-build \
+        rsync \
+        xz
```

Comment:
This is hard to copy&paste and unnecessarily long. Can you please reformat this to just use two or three lines?